app/controllers/assessor_interface/review_verifications_controller.rb

```diff
@@ -11,6 +12,8 @@ def index
       @professional_standing_request =
         assessment.professional_standing_request if assessment.professional_standing_request.verify_failed?
 
+      @teaching_authority_name = region_teaching_authority_name(@assessment.application_form.region, "Contact relevant competent authority")
```

Comment:
One of the benefits of helpers is that we can use them directly in the views, it means all the presentation logic (i.e. how things look on the page) can kept in the view rather than in the view and the controller.